src/react-components/banner/Banner.js

```diff
@@ -0,0 +1,321 @@
+        <div className={styles.branding_container}>
+          <h2>
+            <FormattedMessage defaultMessage="Join the next evolution of Hubs!" id="banner.title" />
+          </h2>
+          <p>
+            <FormattedMessage defaultMessage="Be the first to get a sneak peek!" id="banner.subtitle" />
+          </p>
+        </div>
+
+        <Button
+          preset="primary"
+          onClick={() => {
+            setIsExpanded(true);
+          }}
+        >
+          <FormattedMessage defaultMessage="Find out More" id="banner.expand-cta" />
+          <span className={styles.down_arrow}>
+            <ArrowIcon />
+          </span>
+        </Button>
+      </div>
+
+      {/* EXPAND CONTAINER  */}
+      {isExpanded ? (
+        <div className={styles.expand_wrapper}>
+          <div className={styles.expand_container}>
+            <div className={styles.expand_header}>
+              <h2 className={styles.expand_title}>
+                <FormattedMessage defaultMessage="Join the next evolution of Hubs!" id="banner.expand_title" />
+              </h2>
+              <CloseButton
+                onClick={() => {
+                  setIsExpanded(false);
+                }}
+              />
+            </div>
+
+            <div className={styles.expand_contents}>
+              {/* MESSAGING  */}
+              <div className={styles.expand_messaging}>
+                <FormattedMessage
+                  defaultMessage="{message}"
+                  id="banner.expand-body"
+                  values={{
+                    message:<>We&apos;re working on a new service that makes it easier than ever to deploy a Hub of your own. Sign up here to be the first to know about our new service, as well as the latest Hubs news, product features and offerings. We can&apos;t wait to show you what we&apos;ve been working on!</>
+                  }}
+                />
+              </div>
+
+              {/* FORM  */}
+              <form className={styles.expand_form} onSubmit={handleSubmit(onSubmit)}>
+                {submitted ? (
+                  <Messaging status={responseStatus} />
+                ) : (
+                  <div>
+                    <TextInputField
+                      ref={register}
+                      name="email"
+                      type="email"
+                      label={<FormattedMessage id="email-address" defaultMessage="Email Address" />}
+                      required
+                      value={email}
+                      onChange={onChangeEmail}
+                      placeholder="example@example.com"
+                      className={styles.expand_form_field}
+                    />
+
+                    <RadioInputField className={styles.expand_form_field} label="Format">
+                      <RadioInputOption name="email_format" value="html" label="HTML" ref={register} />
+                      <RadioInputOption name="email_format" value="text" label="Text" ref={register} />
```

Comment:
I think you can just use `<FormattedMessage />` for the values of the `label` attributes, like you did for "Email Address" above.